docs/cli.rst

```diff
@@ -80,13 +80,16 @@ section.
      - URL for the GitLab server
    * - ``private_token``
      - Your user token. Login/password is not supported.
+       Please refer `the official documentation`__ to learn how to obtain a token.
```

Comment:
You can remove `Please` here as you're not asking for permission to the reader ;)

I'm not a native english speaker but I believe "Refer to the official doc..." is better.